<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="listWidget_server">
   <property name="geometry">
    <rect>
     <x>629</x>
     <y>139</y>
     <width>511</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_client">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>139</y>
     <width>511</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_tips">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>1121</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_put">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上传-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_get">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;-下载</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_link" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1121</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,2,0,2,0,2,0,1,0">
    <item>
     <widget class="QLabel" name="label_user">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_user"/>
    </item>
    <item>
     <widget class="QLabel" name="label_user_pass">
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pass">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ip">
      <property name="text">
       <string>ip地址：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ip"/>
    </item>
    <item>
     <widget class="QLabel" name="label_port">
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_port"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_link">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cli_path">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ClientPath</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_srv_path">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ServerPath</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_entry_srv">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Entry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_entry_cli">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>101</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Entry</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
